### ISSUE TITLE
Introduce ADHD mode 

### ISSUE THREAD

**Issue by pritipsingh** [pritipsingh/The-Productive-Champion#2]
May be an overlay that lets you focus on only one part of the screen and rest of the screen has a black opaque screen. We can also make it variable height

**Comment by Ydixit007** [comment]
We can add a black overlay, who's position can be set anywhere on the browser screen. It can have a variable height and width. we can also store the position and size for each website it was previously used on.

**Comment by pritipsingh** [comment]
Yes that's the plan @Ydixit007 , it's same as focus mode. Except this will be applied in all the websites. so a toggle besides a focus mode and then send that value in the backgroun.js, set it in local storage and if the value is true, we can run a script which creates an overlay. Would you be interested in working on it?

**Comment by Ydixit007** [comment]
Sure! I am interested.

**Comment by pritipsingh** [comment]
oops, I missed it. Are you working on it by chance? I am assigning it to you, feel free to change it if not interested anymore.

**Comment by pritipsingh** [comment]
Hey @Ydixit007 by any chance you're still working on this? If not, I am free this week to take it up :)

**Comment by ahsanfarooq210** [comment]
Hi @pritipsingh. Are you working on this feature? I am interested in it
